src/Symfony/Component/HttpClient/Response/StreamWrapper.php

```diff
@@ -178,6 +178,30 @@ public function stream_read(int $count)
         return '';
     }
 
+    public function stream_set_option(int $option, int $arg1, ?int $arg2): bool
+    {
+        if (null === $this->handle || 'stream' !== get_resource_type($this->handle)) {
+            trigger_error(sprintf('The "$handle" property of "%s" needs to be a stream.', __CLASS__), E_USER_WARNING);
+
+            return false;
+        }
+
+        switch ($option) {
+            case STREAM_OPTION_BLOCKING:
+                return stream_set_blocking($this->handle, $arg1);
+            case STREAM_OPTION_READ_TIMEOUT:
+                trigger_error(sprintf('Modifying the timeout after starting the stream is not supported by the StreamWrapper.'), E_USER_WARNING);
```

Comment:
PHP will trigger fatal error because we return false. But I think more precision is a good thing.